hc/api/models.py

```diff
@@ -49,6 +49,7 @@ class Meta:
     timeout = models.DurationField(default=DEFAULT_TIMEOUT)
     grace = models.DurationField(default=DEFAULT_GRACE)
     n_pings = models.IntegerField(default=0)
+    previous_ping = models.DateTimeField(null=True,blank=True)
```

Comment:
What's the difference between `last_ping` and `previous_ping`?

Reply:
`previous_ping` is the time it pinged before the current `last_ping` value, so, when a check pings, I get its `last_ping` value and store it in the `previous_ping` before updating its `last_ping` value, that way I can easily subtract `last_ping` from `previous_ping` to determine whether it has pinged many times than it should